src/viewer.js

```diff
@@ -1608,6 +1611,20 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
         return this.world.removeItem(drawer);
     },
 
+    /**
+     * Update pixel density ration, clears all tiles and triggers updates for
+     * all items.
+     */
+    updatePixelDensityRatio: function() {
```

Comment:
I don't think we need to add this to our public API. The style we're using to denote private functions these days is to add an `_` prefix to the function name. Also add `@private` to the doc comment.

Reply:
I agree. I have fixed it.